bot/exts/utilities/bookmark.py

```diff
@@ -25,6 +23,53 @@
 )
 
 
+class LinkTargetMessage(discord.ui.View):
+    """The button that relays the user to the bookmarked message."""
+
+    def __init__(
+        self,
+        target_message: discord.Message
```

Comment:
Let's add a trailing comma if we choose to use this paranthesis syntax
```suggestion
        target_message: discord.Message,
```

Reply:
I considered that but to be consistent with how they make the init currently, I left it out. And I am not sure what you mean by paranthesis syntax

Reply:
I mean we don't really need to seperate this into multiple lines, line below is 56 chars:
```python
    def __init__(self, target_message: discord.Message):
```